app/src/main/java/org/dicekeys/app/viewmodels/ListDiceKeysViewModel.kt

```diff
@@ -10,8 +10,15 @@ import javax.inject.Inject
 
 @HiltViewModel
 class ListDiceKeysViewModel @Inject constructor(private val encryptedStorage: EncryptedStorage, private val diceKeyRepository: DiceKeyRepository) : ViewModel() {
+
+    fun isDiceKeyInMemory(encryptedDiceKey: EncryptedDiceKey) = diceKeyRepository.exists(encryptedDiceKey)
+
+    fun getDiceKey(encryptedDiceKey: EncryptedDiceKey)  = diceKeyRepository.get(encryptedDiceKey.keyId)
+
     fun remove(encryptedDiceKey: EncryptedDiceKey){
+        // Remove from Storage
```

Comment:
In what context would the user ask to remove a DiceKey from both encrypted long-term storage and memory at once?